js/RecentDonations.js

```diff
@@ -0,0 +1,63 @@
+
+var app = angular.module('RecentDonations', []);
+app.controller('RecentDonationsControl', function($scope) {
+    
+
+    GLOBALS.GET_PARTICIPANT_INFO(160204, function(data) {
+    	$scope.displayName = data["firstName"]
+    })
+
+    $scope.contributors = [{
+    				name:"...", 
+    				amount:"...",
+    				message:"...",
+    				date:"..."}]
+
+    $scope.update = function() {
+    	GLOBALS.GET_PARTICIPANT_DONATION_INFO(160204, function(data) {
+
+    		// Success, hide any error message
+    		$scope.error = undefined
+
+    		// Clear the array
+    		$scope.contributors = []
+
+    		// Go through the array of contributors, and add them to our array
+    		for (var i = 0; i < data.length; i++) {
+
+    			name = data[i].donorName
+    			amount = "$" + data[i].donationAmount
```

Comment:
data[i].donationAmount.toFixed(2), since people could donate in non-whole-dollar amounts.